www/versioned_docs/version-9.x/main/quickstart.mdx

```diff
@@ -23,7 +23,7 @@ For making typesafe API calls from your client. Install in your client codebase
 
 `npm install @trpc/react react-query@3`
 
-For generating a powerful set of React hooks for querying your tRPC API. Powered by [react-query](https://react-query.tanstack.com/).
+For generating a powerful set of React hooks for querying your tRPC API. Powered by [react-query](https://tanstack.com/query/v4/docs/react/overview).
```

Comment:
iirc trpc v9 used react query 3 until the end, see the install commands here: https://trpc.io/docs/v9/quickstart

maybe it makes sense to link to react query v3 in the trpc v9 docs?

(this applies to all links in versioned_docs)

Reply:
Yep! V9 is on v3 so makes sense docs point to v3 here

Reply:
Fixed!